[4/4] Documentation for DeepEval scorers (#19409)
mlflow/mlflow: docs/docs/genai/eval-monitor/scorers/third-party.mdx

import { APILink } from "@site/src/components/APILink";
import TileCard from '@site/src/components/TileCard';
import TilesGrid from '@site/src/components/TilesGrid';
import { Bot, GitBranch, Shield } from "lucide-react";

# Third-party Scorers

> MLflow integrates with popular third-party evaluation frameworks, allowing you to leverage their specialized metrics within MLflow's evaluation workflow. This provides access to battle-tested evaluation metrics while maintaining a consistent MLflow interface.

Reviewer: We don't know how battle-tested 3p framework judges are. Could we just say "access to third party library's evaluation metrics while ..."

## DeepEval Integration

> [DeepEval](https://docs.confident-ai.com/) is a comprehensive evaluation framework for LLM applications that provides metrics for RAG systems, agents, conversational AI, and safety evaluation. MLflow's DeepEval integration allows you to use nearly any DeepEval metric as an MLflow scorer.

Reviewer: ```suggestion
[DeepEval](https://docs.confident-ai.com/) is a comprehensive evaluation framework for LLM applications that provides metrics for RAG systems, agents, conversational AI, and safety evaluation. MLflow's DeepEval integration allows you to use most DeepEval metrics as MLflow scorers.
```
